File: __iOS_BACKUP__/ShapeByte/features/workout/schedule/data/WorkoutScheduleEntry.swift
//
//  File.swift
//  ShapeByte
//
//  Created by Lang, Stefan [Shape Byte Tech] on 02.08.24.
//

import Foundation

struct WorkoutScheduleEntry: Equatable {
    let id: String
    let name: String
    let date: Date
    let progress: Progress
}

File: features/workout/api/src/commonMain/kotlin/de/stefan/lang/shapebyte/features/workout/api/schedule/WorkoutScheduleEntry.kt
package de.stefan.lang.shapebyte.features.workout.api.schedule

import de.stefan.lang.coreutils.api.progress.Progress
import kotlinx.datetime.Instant

data class WorkoutScheduleEntry(
    val id: String,
    val name: String,
    val date: Instant,
    val progress: Progress,
)
